src/components/Announcements/AnnouncementModal.vue

```diff
@@ -0,0 +1,115 @@
+<template>
+  <transition name="modal-fade">
+    <div v-show="open" class="modal-backdrop">
+      <div class="modal">
+        <div class="announce-header">
+          <div class="announce-title">{{announcement.title}}</div>
+          <div class="announce-date">{{ toStringDate(announcement.created) }}</div>
+        </div>
+        <div class="announce-body">{{announcement.description}}</div>
+        <div
+            class="close-btn"
+            @click="close"
+            >
+          X
+        </div>
+      </div>
+    </div>
+  </transition>
+</template>
+
+<script>
+import DateUtils from '../../utils/DateUtils';
+
+export default {
+  name: 'AnnouncementModal',
+  props: {
+    open: {
+      type: Boolean,
+      required: true,
+    },
+    announcement: {
+      type: Object,
+    },
+  },
+  methods: {
+    close() {
+      this.$emit('on-close-announcement');
+    },
+    toStringDate(date) {
+      return DateUtils.toStringDate(date);
+    },
+  },
+};
+</script>
+
+<style scoped>
+  .modal-fade-enter,
+  .modal-fade-leave-active {
+    opacity: 0;
+  }
+  .modal-fade-enter-active,
+  .modal-fade-leave-active {
+    transition: opacity .5s ease
+  }
+
+  .modal-backdrop {
+    position: fixed;
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
+    background-color: rgba(0, 0, 0, 0.3);
```

Comment:
Could be cool to make this a grey. Also this takes up the whole screen, not sure if you want it do that. Also, this a cool package that closes modals when you click outside of it: https://www.npmjs.com/package/vue-clickaway

Reply:
The alpha make it gray transparent like in the screenshot which I was going for. Can definitely address the click away thing later but probably not worth bringing in a new dependency atm